pylabrobot/liquid_handling/liquid_handler.py

```diff
@@ -1760,15 +1761,18 @@ async def dispense96(
       if not len(containers) == 96:
         raise ValueError(f"dispense96 expects 96 wells, got {len(containers)}")
 
-      for channel, well in zip(self.head96.values(), containers):
-        # even if the volume tracker is disabled, a liquid (None, volume) is added to the list
-        # during the aspiration command
-        liquids = channel.get_tip().tracker.remove_liquid(volume=volume)
-        reversed_liquids = list(reversed(liquids))
-        all_liquids.append(reversed_liquids)
+      for well, channel in zip(containers, self.head96.values()):
+        # check if volume tracking is disabled
+        if well.tracker.is_disabled or not does_volume_tracking():
+          reversed_liquids = [(None, volume)]
+          all_liquids.append(reversed_liquids)
+        else:
+          liquids = channel.get_tip().tracker.remove_liquid(volume=volume)
+          reversed_liquids = list(reversed(liquids))
+          all_liquids.append(reversed_liquids)
```

Comment:
liquids (or None if volume tracking is disabled) should have been added in aspiration. That means we can still rack all_liquids. This is needed to make the `MultiHeadDispensePlate` object.

we should keep the change where liquids are only added to the well tracker if liquid tracking is enabled.

Reply:
the reason it's needed in `MultiHeadDispensePlate` is backends can use this information to change liquid handling parameters based on the type of liquid. `None` liquids mean unknown.